### PR TITLE
Take alphas from SCETlib by default, small reorganisation of scripts

### PR DESCRIPTION
**Pull request by kdlong**
- Add main functions and possibility to call scripts with python dict to make it easier to write bias scan scripts
- Use SCETlib alphas as default, because MiNNLO doesn't have the right shape at low pTV

### CODE REVIEW

**Comment by kdlong** [comment]
I forgot, another thing I did here was swap --hdf5 to --root so that hdf5 output is default. Any opposition to that? I need to fix it a few places in the CI if we agree to make the change.

**Comment by bendavid** [comment]
No objections from my side (but even better would be to consistently make the ci use the hdf5 output)

File: utilities/common.py [hunk 1]
```diff
@@ -291,7 +291,7 @@ def __call__(self, parser, namespace, values, option_string=None):
     parser.add_argument("-p", "--postfix", type=str, help="Postfix for output file name", default=None)
     parser.add_argument("--forceDefaultName", action='store_true', help="Don't modify the name of the output file with some default strings")
     parser.add_argument("--theoryCorr", nargs="*", type=str, action=NoneFilterAction,
-        default=["scetlib_dyturbo", ], choices=theory_corrections.valid_theory_corrections(), 
+        default=["scetlib_dyturbo", "scetlib_dyturboCT18ZVars", "scetlib_dyturboCT18Z_pdfas"], choices=theory_corrections.valid_theory_corrections(),
```
Comment (bendavid) [inline utilities/common.py:294]: This means that the scetlib-dyturbo CT18Z pdf variations are loaded (but not necessarily used in the final fit?)

Comment (kdlong) [reply utilities/common.py:294]: Yeah, wasn't sure if I wanted to make that default or not. But probably it doesn't hurt much?

**Comment by bendavid** [comment]
Closing in favor of https://github.com/WMass/WRemnants/pull/463

**Pull request closed without merging**